examples/tutorials/advanced/websockets-example-MNIST/run_websocket_client.py

```diff
@@ -121,7 +120,7 @@ def train(model, device, federated_train_loader, lr, federate_after_n_batches):
     counter = 0
 
     while True:
-        logger.debug("Starting training round, batches [%s, %s]", counter, counter + nr_batches)
+        logger.debug("Starting training round, batches [{}, {}]".format(counter, counter + nr_batches))
```

Comment:
why the use of .format()?

Reply:
@midokura-silvia no particular reason to me it was more pythonic approach also these in .format approach you don't need to specify any type I can change it back if you think there is no need for change